packages/driver/src/cy/commands/actions/type.ts

```diff
@@ -173,7 +173,15 @@ export default function (Commands, Cypress, cy, state, config) {
     const win = state('window')
 
     const getDefaultButtons = (form) => {
-      return form.find('input, button').filter((__, el) => {
+      const formId = form.attr('id')
```

Comment:
I think we need to `CSS.escape` this to avoid issues with special characters breaking the selector:

```suggestion
      const formId = CSS.escape(form.attr('id'))
```